java/BUILD.bazel

```diff
@@ -207,3 +207,28 @@ genrule(
     local = 1,
     tags = ["no-cache"],
 )
+
+java_binary(
```

Comment:
Can you change the main_class to an empty string, and add this comment:
```
This rule is used to package all Ray Java code and the third-party dependencies into a fat jar file. It's not really an executable jar. So we set its `main_class` to empty.
```

Reply:
fixed